docs/competition/team-support/tinker-time.md

```diff
@@ -1,13 +1,23 @@
 # Tinker Time
 
-Tinker Time is time allocated in the schedule to allow teams to test their robot in the arena outside of a match.
+Tinker Time is time allocated in the schedule to allow teams to test their robot in the actual arena.
 
-During Tinker Time up to 3 members of a team may enter the arena.
+Each 'slot' is usually 10 minutes, depending on how much time is scheduled and how many teams are expected. For each slot, team may book a single corner (without preference). Teams should be considerate of other teams and robots in the arena, using the space around the arena to watch rather than all trying to be in the arena at once. Shepherds will not be used to fetch teams for tinker time slot - they will need to remember.
```

Comment:
```suggestion
Each 'slot' is usually 10 minutes, depending on how much time is scheduled and how many teams are expected. For each slot, a team may book a single corner (without preference). Teams should be considerate of other teams and robots in the arena, using the space around the arena to watch rather than all trying to be in the arena at once. Shepherds will not be used to fetch teams for tinker time slot - they will need to remember.
```